src/main/kotlin/no/skatteetaten/aurora/boober/service/resourceprovisioning/ExternalResourceProvisioner.kt

```diff
@@ -56,9 +56,10 @@ class ExternalResourceProvisioner(
             val volume = deploymentSpec.volume ?: return listOf()
 
             val secretVaultNames = volume.mounts?.mapNotNull { it.secretVaultName }.orEmpty()
+            val secretVaultKeys = volume.secretVaultKeys?.mapNotNull { it }.orEmpty()
```

Comment:
Jeg syntes kanskje at volume.secretVaultKeys burde være filtrert på null og evt. satt til empty ved innlesning av auroraConfigFields (og at dette er litt seint; "fail" fast).

Reply:
vet ikke om jeg skjønner helt..
Gjøre om slik at feltet ikke er nullable mener du?

Reply:
Ja - kanskje litt krøkkete forklart. Siden secretVaultKeys i praksis brukes som non-nullable (og filtrert for null-verdier) der hvor den brukes, mener jeg at den bør settes til dette ved initialisering av verdien.